Runtime error in stickify.py when enabling debug log

### What happened?

```
            result = FixedTiledLayout(
                output.device, output.dtype, output.size, output.stride, stl
            )
    
            if logger.isEnabledFor(logging.DEBUG):
                input_info = ", ".join(
                    [f"in{i}:{list(arg.layout.size)}" for i, arg in enumerate(args)]
                )
                logger.debug(
>                   f"{op.__name__} layout: {input_info} -> out:{list(result.size)}, "
                       ^^^^^^^^^^^
                    f"device_size={list(result.device_layout.device_size)}"
                )
E               torch._inductor.exc.InductorError: AttributeError: 'ComputedBuffer' object has no attribute '__name__'
E               
E               Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you're reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS="+dynamo"

../torch_spyre/_inductor/stickify.py:398: InductorError
=========================== short test summary info ============================
FAILED inductor/test_building_blocks.py::TestBuildingBlocks::test__simple_attn
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
============================== 1 failed in 28.72s ==============================
```

### What did you expect to happen?

There should be no error
This is likely due to #1511 


### How can we reproduce it?

Run tests with `SPYRE_INDUCTOR_LOG=1 SPYRE_INDUCTOR_LOG_LEVEL=DEBUG`

### Any environmental details we need to know?

No

### Anything else we need to know?

_No response_

### Relevant log output

```Shell

```

--- a/torch_spyre/_inductor/stickify.py
+++ b/torch_spyre/_inductor/stickify.py
@@ -395,7 +395,7 @@ def pointwise_layout(
                 [f"in{i}:{list(arg.layout.size)}" for i, arg in enumerate(args)]
             )
             logger.debug(
-                f"{op.__name__} layout: {input_info} -> out:{list(result.size)}, "
+                f"{getattr(op, '__name__', repr(op))} layout: {input_info} -> out:{list(result.size)}, "
                 f"device_size={list(result.device_layout.device_size)}"
             )
 

